### PR TITLE
Added support for Norwegian bank account numbers

### PR DESCRIPTION
**Pull request by thor**
This includes *tests* that cover it up fairly well, as well as some documentation to go along.
I `makemessages -l nb` and `compilemessages -l nb`, but if you import from Transifex either way, then I can do the changes there. 🌞 

I haven't been able to figure out exactly what the roadmap is; while it seemed to be a link with Django's roadmap, it also seemed like you're free to release when you've got a worthwhile release, and so I tossed it on a hope of 1.4.2. Admittedly I can change that to 1.5 if need be. 👍 

**All Changes**

- [x] Add an entry to the docs/changelog.rst describing the change.

- [X] Add an entry for your name in the docs/authors.rst file if it's not
      already there.

- [X] Adjust your i ports to a standard form by running this command:

      `isort --recursive --line-width 120 localflavor tests`

**New Fields Only**

- [X] Prefix the country code to all fields.

- [X] Field names should be easily understood by developers from the target
      localflavor country. This means that English translations are usually
      not the best name unless it's for something standard like postal code,
      tax / VAT ID etc.

- [X] Prefer '<country code>PostalCodeField' for postal codes as it's
      international English; ZipCode is a term specific to the United
      States postal system.

- [X] Add meaningful tests. 100% test coverage is not required but all
      validation edge cases should be covered.

- [X] Add `.. versionadded:: <next-version>` comment markers to new
      localflavors.

- [X] Add documentation for all fields.

### CODE REVIEW

File: localflavor/no/forms.py [hunk 1]
```diff
@@ -98,6 +98,66 @@ def _get_birthday(self, value):
...
+        except ValueError:
+            raise ValidationError(self.error_messages['invalid'])
+
+        # They only have one length: the number is 10!
+        # That being said, you always store them with the check digit included.
+        if len(bank_number) is not 10:
+            raise ValidationError(self.error_messages['invalid_length'])
+
+        # Adhere for the special rule of these being "clearing" accounts
+        if bank_number[4:5] == "00":
```
Comment (benkonrath) [inline localflavor/no/forms.py]: `bank_number[4:5]` can never be 2 characters. Did you mean to write `bank_number[4:6]`? I found this because I noticed that line 145 isn't covered by the tests.

**Comment by benkonrath** [comment]
Please also add a changelog entry for this PR. Thanks.

File: localflavor/no/forms.py (hunk 1 above)
Comment (thor) [inline localflavor/no/forms.py]: Thanks for catching that!
I've looked into the clearing numbers, and it's been a struggle to find any more details about exactly how to validate these, so I'm removing it and adding a note about bank clearing numbers in the function docstring.

**Comment by thor** [comment]
I've updated with a less messy version that does the same thing without overriding `Field.clean()` itself.

I've updated the `versionadded`, as well as added the changelog entry for 1.5. 👍 
Furthermore, I've also removed the translations -- do you think you could add a note about that to CONTRIBUTING, or someplace similar?

@benkonrath Extremely so. IBANs are only used in the context of international transactions, and never otherwise referred to domestically.

File: localflavor/no/forms.py [hunk 2]
```diff
@@ -98,6 +97,68 @@ def _get_birthday(self, value):
...
+        # The checksum is 0 in the event there's no remainder, seeing as we cannot have a checksum of 11
+        # when 11 is one digit longer than we've got room for
+        checksum = 0 if remainder is 0 else 11 - remainder
+
+        if checksum != check_digit:
+            raise ValidationError(self.error_messages['invalid_checksum'])
+
+    def to_python(self, value):
+        value = super(NOBankAccountNumber, self).to_python(value)
+        return value.replace(".", "").replace(" ", "")
```
Comment (benkonrath) [inline localflavor/no/forms.py]: Can you change the double quotes to single quotes just to be consistent with the rest of the file?

File: localflavor/no/forms.py [hunk 3]
```diff
@@ -98,6 +97,68 @@ def _get_birthday(self, value):
...
+
+    def prepare_value(self, value):
+        return "{}.{}.{}".format(value[0:4], value[4:6], value[6:11])
```
Comment (benkonrath) [inline localflavor/no/forms.py]: I think it's worth adding a test for the formatting.

Comment (thor) [inline localflavor/no/forms.py]: I've added a simple test to validate it. 👍

File: localflavor/no/forms.py (hunk 2 above)
Comment (thor) [inline localflavor/no/forms.py]: Sure!

**Comment by benkonrath** [comment]
Thanks again for your contribution!

**Comment by thor** [comment]
Thanks for the helpful feedback! 👍

**Comment by thor** [comment]
@benkonrath Hi Ben! Sorry for bumping this, but I'm wondering when you're looking to release a new version? I feel like releasing small and often might be preferable to releasing only with new Django releases, or similarly. :)

**Comment by benkonrath** [comment]
@thor No problem. Thanks for checking in. I was actually planning to do a release this week but other commitments got in the way. I make sure I get it out next week.
